lib/stores/GCSDataStore.js

```diff
@@ -92,7 +92,9 @@ class GCSDataStore extends DataStore {
 
             const file = new File(file_id, upload_length, upload_defer_length, upload_metadata);
             const gcs_file = this.bucket.file(file.id);
+            const parsedMetadata = this._parseMetadataString(upload_metadata);
             const options = {
+                contentType: parsedMetadata.type.decoded,
```

Comment:
tus-js-client uses the `filetype` metadata key to specify the file's type (see https://github.com/tus/tus-js-client#example). I would like it if we only had one property to do so. Would you mind changing it?
Besides that, a check to see if `parsedMetadata.type` actually exists might be necessary.

Reply:
@Acconut small reminder about my question:
>  was going to write it but I couldn't get the GCS test suite to run locally. It does run on travis. I may be missing some env variables. Are there a few steps to follow before running them locally?

Reply:
I responded to it in the main thread two days ago: https://github.com/tus/tus-node-server/pull/130#issuecomment-422711001